stencil/src/components/pulumi-api-doc-filterable-nav/pulumi-api-doc-filterable-nav.tsx

```diff
@@ -33,10 +33,17 @@ export class PulumiApiDocFilterableNav {
         this.parsedNodes = JSON.parse(this.nodes);
         // Before the user interacts with the filter, the nodes to render should be the full nav tree.
         this.currentlyRenderedNodes = this.parsedNodes;
+        this.docsDir = `registry/packages/${this.packageName}/api-docs/`;
```

Comment:
Would it be possible to pass this in as an input to this component as well? I see that it is an input to the `pulumi-api-doc-nav-tree` component already. It would help with making sure that the link always stays up-to-date if the docs were to be relocated in the future.

Reply:
I couldn't find a way to do that with the variables available in Hugo, but am certainly a hugo novice, so if you have any suggestions I agree that would be preferable.